sky/engine/core/script/dom_dart_state.h

```diff
@@ -5,50 +5,28 @@
 #ifndef SKY_ENGINE_CORE_SCRIPT_DOM_DART_STATE_H_
 #define SKY_ENGINE_CORE_SCRIPT_DOM_DART_STATE_H_
 
-#include <memory>
-
-#include "dart/runtime/include/dart_api.h"
+#include "sky/engine/bindings/flutter_dart_state.h"
 #include "sky/engine/platform/fonts/FontSelector.h"
-#include "sky/engine/tonic/dart_state.h"
 #include "sky/engine/wtf/RefPtr.h"
-#include "sky/engine/wtf/text/WTFString.h"
 
 namespace blink {
 class Window;
 
-class DOMDartState : public DartState {
+class DOMDartState : public FlutterDartState {
  public:
-  DOMDartState(std::unique_ptr<Window> window, const std::string& url);
+  DOMDartState(IsolateClient* isolate_client, const std::string& url,
+               std::unique_ptr<Window> window);
   ~DOMDartState() override;
 
-  virtual void DidSetIsolate();
-
   Window* window() const { return window_.get(); }
-  const std::string& url() const { return url_; }
 
   static DOMDartState* Current();
 
-  // Cached handles to strings used in Dart/C++ conversions.
-  Dart_Handle x_handle() { return x_handle_.value(); }
-  Dart_Handle y_handle() { return y_handle_.value(); }
-  Dart_Handle dx_handle() { return dx_handle_.value(); }
-  Dart_Handle dy_handle() { return dy_handle_.value(); }
-  Dart_Handle value_handle() { return value_handle_.value(); }
-  Dart_Handle color_class() { return color_class_.value(); }
-
   void set_font_selector(PassRefPtr<FontSelector> selector);
   PassRefPtr<FontSelector> font_selector();
 
  private:
   std::unique_ptr<Window> window_;
```

Comment:
Renaming will affect a bunch of files throughout the engine.  I'll do this in another PR.